N24DECE089/buoi8/cau6.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+#include <string>  
+using namespace std ; 
+int main()
+{
+    string a , b  ;
+    cout << "moi nhap chuoi " ; getline(cin,a) ;
+    int c = a.size() ; 
+    for ( int i = 0 ; i < a.size() ; ++i ) 
+    { 
+        if ( a[i] != ' ') { b += a[i]; }
+        if ( a[i] == ' ') { cout << b << endl ; b = "" ; }
+        if ( i == c - 1 ) { cout << b ; }
+    }
+    return 0 ; 
+}
```

Comment:
In ra giống như output nha [“Cau”,“lac”, “bo”, “ITMC”]. Bài của em vẫn còn in dấu cách nếu có nhiều dấu cách ở giữa các chữ. Chú ý sửa lại nha